### DISCUSSION TITLE
Error from parameter identification in PK-Sim

### DISCUSSION THREAD

**Discussion by teutonicod** [Open-Systems-Pharmacology/Forum#1659] [Questions & Problems]
Hi,

I developed a model with PK-Sim first, then with MoBI. I then imported again the model on PK-Sim and tried to do parameter estimations there. The simulation runs normally, and never gave any error message, but if I try to perform a PI in PK-Sim I get the following error message.

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/11489621/1f9f9d77-f63d-4903-bca3-c167f39e7a91)

Could anyone point me in the direction of the origin of this error message? I tried to change the values of the parameters and algorithms, but the error remains. In MoBi the parameter estimation with the same simulation runs normally.

Thanks a lot for your advice!

Donato

**Comment by StephanSchaller** [comment]
Hi Donato, so it seems the model is not stable, i.e., the solver cannot solve it. 
But, the re-import of MoBi Models to PK-Sim is (only) intended for population simulation.
Nevertheless, it might be due to different solver settings in PK-Sim vs MoBi, did you check if they are the same?

**Comment by teutonicod** [comment]
Hi Stephan,

Thanks for your reply. Actually, you can also import individual simulation in PK-Sim from MoBi, not only for population simulations, and this is what I did.

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/11489621/04ff1438-4bab-4394-a9e8-1ae6bf1f4bc9)

I find simulations easier to handles in PK-Sim, for summary plots and PK analysis.
Anyway, I checked the settings of the parameter identification, and they are identical between MoBi and PK-Sim.
I also checked the solver setting and they are the same as well. 
Any other idea?

Thanks,

Donato

**Comment by Yuri05** [comment]
@teutonicod Can you increase the simulation resolution (e.g. set it to 100 points/**minute**) in the first 3 minutes?
What's the error message then in the PI?
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/ac849fc0-74ae-48fb-9cbe-037920328167)

**Reply by teutonicod** [reply]
Hi @Yuri05 , the error message remain the same, it just shift the time to a smaller time (0.6 now). I also tried a higher number of timepoints in the resolution of the first 3min but the error remains

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/11489621/680cef87-608c-402c-8195-b8be1cb86ec2)

**Reply by Yuri05** [reply]
Hard to say. Is it possible to upload the project or to send it to me?
If not: can you attach the screenshots of the PI configuration (parameter settings, algo settings)?

**Comment by PavelBal** [comment]
The PI fails with the above message if it tests a parameter set that cannot be solved.

Try using the "Monte Carlo" algorithm, it will not fail in this case but will try another parameter values set instead.

**Reply by teutonicod** [reply]
Hi @PavelBal , I was already using the Monte Carlo estimation. I also tried with the other algorithm but I got the same error with all of them.
